Makefile

```diff
@@ -32,3 +32,18 @@ create-stack: requirements lint generate cf-lint
 	  --stack-name unionai-support-stack \
 	  --template-body file://./union-ai-admin/aws/unionai-support-role.template.yaml \
 	  --capabilities CAPABILITY_IAM CAPABILITY_NAMED_IAM CAPABILITY_AUTO_EXPAND
+
+# RELEASE_TAG=v5.1.1 make release_cloudformation
+.PHONY: release_cloudformation
+release_cloudformation:
+	# Create Git Release
+	git tag $(RELEASE_TAG)
+	git push origin $(RELEASE_TAG)
+
+	# Create directory for the new release
+	aws s3api put-object --bucket $(BUCKET_NAME) --key templates/$(RELEASE_TAG)/
```

Comment:
Where does `$BUCKET_NAME` get set?

Reply:
Ah, I see, I think you had a change that didn't get pushed up -- I see it now with https://github.com/unionai/union-cloud-infrastructure/pull/13/commits/687cec58b5b47e19fcf441186ba6f37874bd37dc